hadoop-ozone/dist/src/main/compose/ozone-ha/docker-compose.yaml

```
@@ -103,7 +103,7 @@ services:
   httpfs:
     <<: *common-config
     environment:
-      OZONE_CLASSPATH: "/opt/hadoop/share/ozone/lib/ozone-filesystem-1.3.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ozone-filesystem-common-1.3.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ozone-filesystem-hadoop3-1.3.0-SNAPSHOT.jar"
+      OZONE_CLASSPATH: "/opt/hadoop/share/ozone/lib/ozone-filesystem-${ozone.version}.jar:/opt/hadoop/share/ozone/lib/ozone-filesystem-common-${ozone.version}.jar:/opt/hadoop/share/ozone/lib/ozone-filesystem-hadoop3-${ozone.version}.jar"
```

The `ozone-filesystem-hadoop3` jar has all classes from the other two jars, it can be used in itself.

```suggestion
      OZONE_CLASSPATH: "/opt/hadoop/share/ozone/lib/ozone-filesystem-hadoop3-${ozone.version}.jar"
```

---

thanks Attila, I updated my patch accordingly!